com.github.tenderowl.norka.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "com.github.tenderowl.norka",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.tenderowl.norka",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
```

Comment:
What does it use the home access for? If it's just to save the documents, you can use Gtk.FileChooserNative for that,  which uses the portals and you won't need a filesystem access anymore.

Reply:
Access to the home folder was added for export and import files via drag-n-drop. And as I understood flatpak docs it is required to place documents database file into the user's home directory.
I'm not sure it will work fine in ElementaryOS, probably I should check for it.

Reply:
It should work just fine then :) this is not a blocker, but rather a suggestion to improve the sandboxing of the application.

Reply:
Oh, I see. Will check for it anyway, thanks!